### PR TITLE
refactor: update remaining http samples to use declarative func signature

### PR DESCRIPTION
**Pull request by KaylaNguyen**
## Description

Update remaining http samples to use declarative func signatures and fix import not in samples.

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by KaylaNguyen** [comment]
Hi there, any update on this?

**Comment by leahecole** [comment]
@engelke can you PTAL or triage to someone else in @GoogleCloudPlatform/aap-dpes ?

File: functions/v2/audit_log/main.py [hunk 1]
```diff
@@ -14,25 +14,29 @@
...
+
 # CloudEvent function to be triggered by an Eventarc Cloud Audit Logging trigger
 # Note: this is NOT designed for second-party (Cloud Audit Logs -> Pub/Sub) triggers!
-def hello_auditlog(cloudevent):
+@functions_framework.cloud_event
+def hello_auditlog(cloud_event):
     # Print out the CloudEvent's (required) `type` property
     # See https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#type
-    print(f"Event type: {cloudevent['type']}")
+    print(f"Event type: {cloud_event['type']}")
```
Comment (ace-n) [inline functions/v2/audit_log/main.py:26]: Nit: is there a reason why we shouldn't use `cloudevent` here?

(I think we still do that in other languages/samples.)

**Review by ace-n — APPROVED** [review]

File: functions/v2/audit_log/main.py (hunk 1 above)
Comment (KaylaNguyen) [reply functions/v2/audit_log/main.py:26]: This is due to the discussion of "Cloud Event" being 2 words so we should use snake case in code for consistency everywhere: https://github.com/GoogleCloudPlatform/functions-framework-python/issues/166

**Comment by KaylaNguyen** [comment]
Can we have someone from @GoogleCloudPlatform/python-samples-owners to look at this change also? Thanks!

**Comment by KaylaNguyen** [comment]
Friendly ping on this :)

**Review by dandhlee — APPROVED** [review]
Sorry for the late review! LGTM.

**Pull request merged**